### ISSUE TITLE
Wont play on Chrome 56.0.2924.87

### ISSUE THREAD

**Issue by anothercoder2** [unit9/oz#2]
Tried with multiple OSs the site wont play with Chrome  56.0.2924.87.
It used to play earlier.

**Comment by silviopaganini** [comment]
yeah, there's a problem with the sound library, probably due to the webaudio been updated

@tjomk do you know if we still have access to the GAE?

**Comment by tjomk** [comment]
@silviopaganini nope, we don't anymore. I'll check if we can get access to it.
